### PR TITLE
HSEARCH-2258 Normalized names conflict

### PR DESCRIPTION
**Pull request by DavideD**
https://hibernate.atlassian.net/browse/HSEARCH-2258

### CODE REVIEW

File: engine/src/main/java/org/hibernate/search/util/logging/impl/Log.java [hunk 1]
```diff
@@ -1000,4 +1000,7 @@
 	@Message(id = 324, value = "The fieldBridge for field '%1$s' is an instance of '%2$s', which does not implement TwoWayFieldBridge. Projected fields must have a TwoWayFieldBridge.")
 	SearchException projectingFieldWithoutTwoWayFieldBridge(String fieldName, Class<?> fieldBridgeClass);
 
+	@Message(id = 325, value = "Index names collision detected. Different logical indexes will point to the same actual index: %1$s")
```
Comment (yrodiere) [inline engine/src/main/java/org/hibernate/search/util/logging/impl/Log.java]: I couldn't find any place where we use the terms "logical indexes". Ok with you if I replace that with "index managers"?

**Comment by DavideD** [comment]
Updated and rebased

**Comment by yrodiere** [comment]
Jenkins, test this please

**Comment by yrodiere** [comment]
Rebased and merged. Thanks!

**Pull request merged**